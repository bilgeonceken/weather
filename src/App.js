import React from 'react';

import { Provider } from 'react-redux';
import store from './store';

import Forecast from './components/Forecast';
import { GlobalStyle } from './styles/global';

const App = () => {
  return (
    <Provider store={store}>
      <div>
        <GlobalStyle />
        <Forecast />
      </div>
    </Provider>
  );
};

export default App;
