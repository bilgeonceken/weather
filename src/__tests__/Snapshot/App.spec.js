import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../MockApp';

describe('watchFetchForecast', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
