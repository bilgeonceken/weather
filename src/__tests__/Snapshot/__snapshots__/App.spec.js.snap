// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`watchFetchForecast renders correctly 1`] = `
<div>
  <div
    className="sc-hKwDye bpuQHS"
  >
    <header
      className="sc-bdvvtL hcnYhO"
    >
      <article
        className="left"
      >
        <img
          alt="Clear Icon"
          src="fileurl"
        />
        <div
          className="temp"
        >
          <div
            className="temp-upper"
          >
            <p
              className="temp-weather"
            >
              Clear
            </p>
            <p
              className="temp-highlow"
            >
              8° / 13°
            </p>
          </div>
          <div
            className="temp-lower"
          >
            0°
          </div>
        </div>
      </article>
      <article
        className="right"
      >
        <div
          className="data"
        >
          <div
            className="data-upper"
          >
            <p
              className="data-location"
            >
              Altstadt
            </p>
          </div>
          <div
            className="data-lower"
          >
            <p>
              Thursday
            </p>
            <p>
              16. February
            </p>
          </div>
        </div>
      </article>
    </header>
    <ul
      className="sc-gsDKAQ boIDyf"
    >
      <div
        className="sc-dkPtRN itRkwX"
        onClick={[Function]}
      >
        <span
          className="timestamp"
        >
          12:00
        </span>
        <img
          alt="Clear Icon"
          src="fileurl"
        />
        <p>
          6°
        </p>
      </div>
      <div
        className="sc-dkPtRN fnkRDf"
        onClick={[Function]}
      >
        <span
          className="timestamp"
        >
          15:00
        </span>
        <img
          alt="Clear Icon"
          src="fileurl"
        />
        <p>
          6°
        </p>
      </div>
    </ul>
  </div>
</div>
`;
